plio/permissions.py

```diff
@@ -14,7 +15,14 @@ def has_permission(self, request, view):
         return True
 
     def has_object_permission(self, request, view, obj):
-        """Object-level permissions for plio. This determines whether the request can access a plio instance or not."""
+        """Object-level permissions for plio/item/question. This determines whether the request can access a plio/item/question instance or not."""
+
+        # if the requested object is an Item or Question, filter the Plio record that it belongs to
+        # and use the permission logic on that Plio instance
+        if isinstance(obj, Question):
+            obj = Plio.objects.filter(id=obj.item.plio.id).first()
+        elif isinstance(obj, Item):
+            obj = Plio.objects.filter(id=obj.plio.id).first()
```

Comment:
was this added just for sanity or did something broke that led you to add this?
the permission class has only been added to `QuestionViewSet` and not `ItemViewSet`-  any reason?
Any reason for not adding this for ImageViewSet too?